apps/worker/src/app/workflow/usecases/message-matcher/message-matcher.usecase.ts

```diff
@@ -321,7 +321,7 @@ export class MessageMatcher extends Filter {
     filterProcessingDetails: FilterProcessingDetails
   ): Promise<boolean> {
     const subscriber = await this.subscriberRepository.findOne({
-      _id: command.subscriberId,
+      _id: command._subscriberId,
```

Comment:
https://discord.com/channels/895029566685462578/1146039495398010982/1166071042213679144

Reply:
Totally right i just now noticed that we still have not merged the deprecation of this use case on behalf of using ConfitionFilter. 🤦‍♂️

The refactor is located here
https://github.com/novuhq/novu/pull/4187